File: app/src/main/java/com/galtashma/parsedashboard/ParseDashboardApplication.java
package com.galtashma.parsedashboard;

import android.app.Application;

import com.appizona.yehiahd.fastsave.FastSave;

public class ParseDashboardApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        FastSave.init(getApplicationContext());
    }
}

File: app/src/main/java/com/galtashma/parsedashboard/ParseDashboardApplication.kt
package com.galtashma.parsedashboard

import android.app.Application
import com.appizona.yehiahd.fastsave.FastSave

/**
 * Created by gal on 3/16/18, rewritten by Cyb3rKo on 05/12/22.
 */

class ParseDashboardApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        FastSave.init(applicationContext)
    }
}
